cmd/remoteva/main.go

```diff
@@ -0,0 +1,130 @@
+package notmain
+
+import (
+	"context"
+	"flag"
+	"os"
+	"time"
+
+	"github.com/letsencrypt/boulder/bdns"
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/features"
+	bgrpc "github.com/letsencrypt/boulder/grpc"
+	"github.com/letsencrypt/boulder/va"
+	vacfg "github.com/letsencrypt/boulder/va/config"
+	vapb "github.com/letsencrypt/boulder/va/proto"
+)
+
+type Config struct {
+	RVA struct {
+		vacfg.Common
+		TLSClient cmd.TLSConfig
+	}
+
+	Syslog        cmd.SyslogConfig
+	OpenTelemetry cmd.OpenTelemetryConfig
+}
+
+func main() {
+	grpcAddr := flag.String("addr", "", "gRPC listen address override")
+	debugAddr := flag.String("debug-addr", "", "Debug server address override")
+	configFile := flag.String("config", "", "File path to the configuration file for this service")
+	flag.Parse()
+	if *configFile == "" {
+		flag.Usage()
+		os.Exit(1)
+	}
+
+	var c Config
+	err := cmd.ReadConfigFile(*configFile, &c)
+	cmd.FailOnError(err, "Reading JSON config file into config structure")
+
+	features.Set(c.RVA.Features)
+
+	if *grpcAddr != "" {
+		c.RVA.GRPC.Address = *grpcAddr
+	}
+	if *debugAddr != "" {
+		c.RVA.DebugAddr = *debugAddr
+	}
```

Comment:
You could lump dedupe all of the common field validations into a `.validate()` method on `vaconfig`.

```suggestion
	if *grpcAddr != "" {
		c.RVA.GRPC.Address = *grpcAddr
	}
	if *debugAddr != "" {
		c.RVA.DebugAddr = *debugAddr
	}
```